internal/pkg/platform/platform.go

```diff
@@ -29,6 +30,7 @@ type Platform interface {
 	UserData() (*userdata.UserData, error)
 	Mode() runtime.Mode
 	Hostname() ([]byte, error)
+	InstanceIPs() ([]net.IP, error)
```

Comment:
nit: Can we call this `ExternalIPs`? I feel like that is a better description here.